tv-shows-app/src/app/components/shared/Icons/StarIcon/StarIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { Icon } from "@chakra-ui/react";
+import { CheckCircleIcon} from '@chakra-ui/icons';
+import { Fragment, useState } from "react";
+
+export interface IStar {
+   selected: boolean
+}
+
+export interface IStarProps {
+   star: IStar
+   onChange: (star : IStar) => void;
+}
+
+
+const urls = ['/images/star-white.png', 'images/star-yellow.png'];
+
+export default function StarIcon({star, onChange} : IStarProps) {
+   const imgUrl = (star.selected) ? urls[1] : urls[0];
+   
+   return <div>

```

Comment:
Možda to nismo naglasili ali s obzirom da smo introducali ChakraUI trudimo se koristiti iskljucivo njene elemente.
Drugim riječima ukoliko nam treba `div`, koristiti ćemo npr. `Box` element.